pkg/apis/storageos/v1/storageoscluster_types.go

```diff
@@ -40,6 +40,7 @@ const (
 	CSIv0DriverRegistrarContainerImage        = "quay.io/k8scsi/driver-registrar:v0.4.2"
 	CSIv0ExternalProvisionerContainerImage    = "storageos/csi-provisioner:v0.4.2"
 	CSIv0ExternalAttacherContainerImage       = "quay.io/k8scsi/csi-attacher:v0.4.2"
+	DefaultNFSContainerImage                  = "storageos/nfs:test"
```

Comment:
Is this a good default or should we use `latest`?

Reply:
It's just for now. Will set it to some proper version before the next release.
I don't see `latest` tag in docker hub. Maybe we can have some pre-release tag for the next release.